go.mod

```diff
@@ -4,18 +4,15 @@ go 1.15
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
-	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
-	github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4 // indirect
 	github.com/envoyproxy/go-control-plane v0.9.8
-	github.com/golang/protobuf v1.4.2
+	github.com/golang/protobuf v1.4.3
```

Comment:
I forget what our policy on go dependency bumps is but we might want to do this in a separate PR? cc @projectcontour/maintainers

Reply:
Generally, we prefer dependency bumps in separate PRs, if this is required for this change though, I think it's okay this time.

Reply:
Filed issue: https://github.com/projectcontour/contour/issues/3335
PR: https://github.com/projectcontour/contour/pull/3336

Reply:
Yes, bunch of stuff is getting changed if I update prometheus golang library.